src/falconpy/sensor_download.py

```diff
@@ -0,0 +1,90 @@
+from ._util import service_request, parse_id_list
+from ._service_class import ServiceClass
+
+import os
+
+
+class Sensor_Download(ServiceClass):
+
+    def GetCombinedSensorInstallersByQuery(self: object, params: dict) -> dict:
+        """
+        retrieve all metadata for installers from provided query
+        """
+        FULL_URL = self.base_url+'/sensors/combined/installers/v1'
+        HEADERS = self.headers
+        PARAMS = params
+        returned = service_request(caller=self,
+                                   method="GET",
+                                   endpoint=FULL_URL,
+                                   params=PARAMS,
+                                   headers=HEADERS,
+                                   verify=self.ssl_verify
+                                   )
+        return returned
+
+    def DownloadSensorInstallerById(self: object, _id: str, file_name: str, download_path: str = "sensor_downloads"):
```

Comment:
```suggestion
    def DownloadSensorInstallerById(self: object, _id: str, file_name: str, download_path: str = None) -> object:
```

Reply:
Curious why you decided to go with a private for _id as opposed to using the ids pattern?  Is it because it's always one ID?

Reply:
flipped in newest commit, been writing personal code calling things like that as no one else uses it :) changed to be id_